src/lib/zkapp.ts

```diff
@@ -1067,7 +1074,7 @@ super.init();
       }
       for (let methodIntf of methodIntfs) {
         let accountUpdate: AccountUpdate;
-        let { rows, digest, result } = analyzeMethod(
+        let { rows, digest, result, gates } = analyzeMethod(
```

Comment:
Should we adjust the doc comment for `analyzeMethod` as well? I am actually not sure what it says right now

Reply:
oh right, will update the comment `SmartContact.analyzeMethods()`